app/services/planning_application_search.rb

```diff
@@ -145,6 +163,10 @@ def query_terms
     @query_terms ||= query.split.join(" | ")
   end
 
+  def address_query_terms
+    @address_query_terms ||= query.split.map { |term| term }.join(" & ")
+  end
+
```

Comment:
I feel like this could be a local variable inside the `records_matching_address` method — it doesn't seem like it's used anywhere else, or used more than once per query — unless I'm missing something?

Reply:
Also I don't think `map { |term| term }` does anything, does it?